source/templates.tex

```diff
@@ -117,7 +117,7 @@
 shall not be a
 \grammarterm{template-id}.
 \begin{note}
-That last component may be an \grammarterm{identifier}, an \grammarterm{operator-function-id},
+That last component could be an \grammarterm{identifier}, an \grammarterm{operator-function-id},
```

Comment:
```suggestion
That last component can be an \grammarterm{identifier}, an \grammarterm{operator-function-id},
```
This is contrasting a "shall not be", so "can be" seems better than "could be" to me.

Reply:
fixed